page.yaml

```diff
@@ -2664,9 +2664,23 @@ groups:
         requirements:
           - go install -v github.com/go-critic/go-critic/cmd/gocritic@latest
       - title: Reference and run common linters
-        description: This tool has comprehensive list of linters. Owners of this aggregator keep track of active linters, their versions, and optimal configs. It contains many optimizations to make linters run fast by paralleism, distributing binaries and Docker images, utilising `golang.org/x/tools/go/analysis` toolchain.
-        url: https://github.com/golangci/golangci-lint
-        name: golangci
+        description: This tool has comprehensive list of linters. Owners of this aggregator keep track of active linters, their versions, and optimal configs. It contains many optimizations to make linters run fast by parallelism, distributing binaries and Docker images, utilising `golang.org/x/tools/go/analysis` toolchain.
+        url: https://golangci-lint.run
+        name: golangci-lint
+        commands:
+          - golangci-lint run
+        requirements:
+          - curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin latest
```

Comment:
I avoid direct shell installers and binaries, rather using `go install` is preferred.

but golangci-lint historically pro-binary, so guess will let people decide for themselves